src/services/ethereum/factory.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import EthereumService from './index';
+import CustomSigner from './CustomSigner';
+
+/**
+ * @desc Factory for the Ethereum service based on EthJS.
+ * @param {Array<any>} config Array with: private_key Private Key base for the services, provider_type Type of Network
```

Comment:
It seems that you actually mean `Object` here instead of `Array<any>`

Reply:
Yes, changed